src/Sentry/Infrastructure/FileDiagnosticLogger.cs

```diff
@@ -30,7 +30,10 @@ public FileDiagnosticLogger(string path, bool alsoWriteToConsole = false)
     public FileDiagnosticLogger(string path, SentryLevel minimalLevel, bool alsoWriteToConsole = false)
         : base(minimalLevel)
     {
+        // Allow direct file system usage
+#pragma warning disable SN0001
```

Comment:
Ah good point, @bitsandfoxes anything we need to do to make this "Not available on Switch" ? 

Edge case since someoen needs to get out of their way to use this and its' just for debugging, but this analyzier really help us make these calls now.

Not doing anything about this (just ignoring it) is fine by me too but we should add a note to the docs that this thing isn't supported on Switch anyway:

https://docs.sentry.io/platforms/unity/configuration/diagnostic-logger/#FileDiagnosticLogger